addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -16,22 +21,146 @@ Chatter.components = {
     FollowerList,
 };
 
-Chatter.props = [...Chatter.props, "has_activities?"];
+Chatter.props = [...Chatter.props, "has_activities?", "hasParentReloadOnFollowersUpdate?"];
 
-Chatter.defaultProps = { ...Chatter.defaultProps, has_activities: true };
+Chatter.defaultProps = {
+    ...Chatter.defaultProps,
+    has_activities: true,
+    hasParentReloadOnFollowersUpdate: false,
+};
 
 patch(Chatter.prototype, {
     setup() {
         super.setup(...arguments);
         this.activityService = useState(useService("mail.activity"));
         this.recipientsPopover = usePopover(RecipientList);
+        this.state = useState({
+            ...this.state,
```

Comment:
I don't think we should do it this way, I don't even know how that would work actually.

Isn't it possible to just add a new key into the existing state?
Otherwise, a new state variable should be used.

Reply:
another option: `this.webState = useState({ new_keys ...` (not a great name)

Reply:
Another option: `this.state = useState(this.getDefaultState()` in parent, then override `getDefaultState`

Reply:
For counter-example of the current diff, in parent:
```js
        const state = this.state;
        setTimeout(() => {
            state.jumpThreadPresent++;
        }, 10000);
        ```

Reply:
Used `Object.assign()` since in the follow up PRs, there will be more keys in the state.